Makefile

```diff
@@ -136,7 +136,7 @@ ifneq ($(HAS_WEBAPP),)
 	mkdir -p dist/$(PLUGIN_ID)/webapp
 	cp -r webapp/dist dist/$(PLUGIN_ID)/webapp/
 endif
-	cd dist && tar -cvzf $(BUNDLE_NAME) $(PLUGIN_ID)
+	cd dist && COPYFILE_DISABLE=1 tar -cvzf $(BUNDLE_NAME) $(PLUGIN_ID)
```

Comment:
We tend not to deviate from our start template, why is this required?

Reply:
`make deploy` fails in ARM Macbooks without this. There is open issue in mattermost repo https://github.com/mattermost/mattermost-server/issues/22614

Reply:
I recall having problems with tar myself. Did you try installing `gnutar` ?  https://formulae.brew.sh/formula/gnu-tar